auth/authRouter.js

```diff
@@ -76,6 +93,23 @@ router.post('/login',
   }
 );
 
+// ********************************************************
+//  auth with google
+// ********************************************************
+
+router.get('/google', passport.authenticate('google', {
+  scope:['profile', 'email']
+}));
+
+
+
+router.get('/google/callback', passport.authenticate('google', { session: false}), (req,res) => {
+  console.log('callback', req.user );

```

Comment:
This can be taken out since presumably the debugging you were using it for worked out and it's no longer needed.